src/main/java/com/example/demo/manage/ManageUser.java

```diff
@@ -64,29 +59,26 @@ public void setUserRepository(UserRepository userRepository) {
 	 */
 	
 
-	@RequestMapping(value = "/id",method = RequestMethod.GET, produces = {MediaType.APPLICATION_JSON_VALUE })
-	public ResponseEntity<String> searchUserById(@RequestParam("id") String id) throws JsonProcessingException {
-		String jsonInString = userService.getsearchUserById(id);
+	@GetMapping(value = "/id", produces = {MediaType.APPLICATION_JSON_VALUE })
+	public ResponseEntity<UserResponseDto> searchUserById(@RequestParam("id") String id) throws JsonProcessingException {
+		Optional<User> user = userService.getsearchUserById(id);
+		UserResponseDto response = userAssembler.toModel(null,user);
```

Comment:
Write separate assembler if there is need and change response dto as well in that case.

Reply:
Written another OptionalUserAssembler for Optional Data Types.
Kept UserDto as response Dto in this